src/encoded/static/img/immune-cells/platelets.svg

```diff
@@ -0,0 +1,34 @@
+<style type="text/css">
+	.st0{fill:#FFFFFF;stroke:#000000;stroke-width:0.5;stroke-miterlimit:10;}
+</style>
+<path class="st0" d="M4,0.5C3.8,0.9,4.5,1.5,4.5,2.3C4.5,3,3.8,3.8,3,4.1C2,4.4,1.1,3.6,0.6,4.1C0.4,4.3,0.3,4.6,0.4,4.8
+	S0.7,5,2.2,5.2c0.3,0,0.5,0.1,0.5,0.2c0.2,0.2-0.2,0.5-0.1,1c0.1,0.5,0.5,0.6,0.5,1c0,0.2-0.3,0.3-0.9,0.8c-0.7,0.5-1,0.8-1,1
+	c0,0.2,0.5,0.4,0.8,0.4c0.3,0,0.4-0.2,1.2-0.8c0.6-0.4,0.7-0.5,0.8-0.5c0.5-0.1,0.7,0.2,1.3,0.2c0.7,0,0.7-0.6,1.3-0.5
+	c0.7,0,1.2,0.7,1.2,0.7C8.2,9.3,8,9.9,8.3,10c0.2,0,0.5-0.3,0.6-0.7C9,8.8,8.5,8.3,8.4,8.1C8,7.8,7.7,7.7,7.7,7.4
+	c0-0.2,0-0.4,0.2-0.5c0.2-0.1,0.4,0,1.3,0.3c0.4,0.1,0.6,0.2,0.8,0.1c0.2-0.1,0.5-0.3,0.5-0.5c0-0.4-1-0.5-1.5-0.6
+	C8,5.9,7.7,6,7.6,5.8C7.3,5.4,8.3,4.5,8,4.1c-0.2-0.2-0.8,0.1-1-0.3c-0.2-0.2,0-0.4-0.2-0.7C6.5,2.7,5.9,2.9,5.5,2.6
+	C5.2,2.4,5.2,1.9,4.9,1C4.9,0.7,4.8,0.5,4.6,0.4C4.4,0.3,4.1,0.4,4,0.5z"/>
+<path class="st0" d="M10.6,1.9c0.2-0.1,0.5,0.1,0.8,0.3C12,2.5,12,2.7,12.5,2.9c0.3,0.2,0.7,0.3,1,0.2C13.7,3,13.6,2.8,14,2.4
+	C14.2,2.2,14.4,2,14.5,2c0.2,0.1,0,0.5,0.3,0.7C15,3,15.3,2.9,15.9,2.9c1,0.1,0.9,0.4,1.6,0.4c0.4,0,0.5-0.2,1.4-0.3
+	c0.7-0.1,0.7,0,0.7,0c0.1,0.1-0.3,0.5-0.7,0.7C18.3,4,18,3.8,17.7,4.1c-0.3,0.3-0.4,0.8-0.4,0.8c0,0.2,0,0.3,0,0.6
+	c0,0.4-0.2,0.5-0.2,0.7c0,0.4,0.2,0.7,0.6,1c0.5,0.6,0.9,0.7,0.9,1c0,0.2-0.3,0.5-0.5,0.5c-0.2,0-0.4-0.2-0.9-0.7
+	c-0.7-0.7-0.9-0.8-1-0.8c-0.3,0-0.3,0.5-0.7,0.7C15.2,8,15,7.8,14.6,8c-0.2,0.1-0.3,0.1-0.4,0.2c-0.1,0.2-0.1,0.3,0,1
+	c0.1,0.7,0.1,0.8,0.1,1c0,0.2-0.2,0.6-0.4,0.7c-0.2,0-0.6-0.6-0.6-1.2c0-0.5,0.2-0.7,0.1-1.2c-0.1-0.4-0.2-0.6-0.3-0.7
+	c-0.3-0.5-0.5-0.4-0.7-0.7c-0.3-0.6,0.2-1-0.2-1.5c-0.3-0.4-0.9-0.6-1.3-0.6c-0.3,0-0.4,0.1-1,0.2C9.4,5.2,9.1,5.3,9,5.2
+	C9,5,9.3,4.7,9.5,4.5c0.4-0.3,0.8-0.4,1-0.4c0.4-0.1,0.6-0.1,1,0c0.2,0.1,0.4,0.2,0.6,0.1c0.1-0.1,0.2-0.2,0.2-0.4
+	c0-0.2-0.3-0.2-0.7-0.4C11.3,3.2,11,3,10.8,2.7C10.5,2.5,10.4,2,10.6,1.9z"/>
+<path class="st0" d="M6.1,13.4c0.1-0.5-0.1-0.7,0.1-1.2C6.2,12.1,6.3,12,6.4,12c0.2-0.1,0.3,0.3,0.6,0.3c0.3,0,0.4-0.3,0.9-0.7
+	c0.4-0.4,0.5-0.2,1.1-0.6c0.3-0.2,0.7-0.5,1-1c0.3-0.4,0.3-0.7,0.6-0.9C10.7,9,11.1,8.9,11.2,9c0.1,0.1,0,0.4-0.3,0.9
+	c-0.3,0.6-0.4,0.7-0.5,1c0,0.1-0.1,0.5,0,0.9c0.1,0.3,0.2,0.3,0.8,1c0.5,0.6,0.5,0.7,0.7,0.8c0.4,0.3,0.9,0.3,1.1,0.3
+	c0.5,0,0.8-0.1,1,0c0.2,0.1,0.2,0.4,0.1,0.6c-0.1,0.2-0.4,0.1-1.3,0.1c-0.9,0-1,0.1-1.1,0.2c-0.2,0.2-0.1,0.3-0.3,0.9
+	c-0.2,0.4-0.3,0.4-0.5,1c-0.2,0.4-0.2,0.6-0.2,0.8c0.1,0.3,0.3,0.4,1,0.9c0.6,0.5,0.7,0.7,0.7,0.8c0,0.3-0.3,0.5-0.5,0.6
+	c-0.4,0.1-0.9-0.3-1.1-0.5c-0.6-0.5-0.4-0.8-0.9-1.2c-0.5-0.5-1.2-0.5-1.3-0.5c-0.5,0-0.6,0.1-0.9,0c-0.4-0.2-0.4-0.5-0.8-0.8
+	c-0.1,0-0.4-0.2-0.8-0.2c-0.7,0.1-1.1,0.9-1.2,1.1c-0.2,0.5-0.2,0.7-0.5,0.9c-0.3,0.2-0.7,0.1-0.8-0.1C3.5,18,4,17.3,4.5,16.8
+	c0.6-0.5,1-0.6,1.1-1c0-0.3-0.2-0.6-0.5-0.8c-0.4-0.2-0.6,0.1-1.3,0.1c-0.4,0-1-0.1-1.1-0.3c0-0.2,0.4-0.5,0.8-0.6
+	c0.4-0.1,0.5,0.1,1.1,0.1c0.4,0,0.7,0,0.9-0.2C6,14,6,13.6,6.1,13.4z"/>
+<path class="st" d="M-2.1,12.5"/>
```

Comment:
Not that this makes more than a few bytes difference, but this path doesn’t render anything.

Reply:
I think we should fix it since this ticket won't be holding up the sprint.